src/entities/http-config.ts

```diff
@@ -7,6 +7,12 @@ export interface HttpConfig {
    * Additional headers to include in all HTTP requests.
    */
   additionalHeaders?: Record<string, string>;
+  /**
+   * Set this property to your proxy URL *only* if you've received a proxy
+   * key value from your RevenueCat contact. This value should never end with
+   * a trailing slash.
+   */
+  proxyURL?: string;
```

Comment:
Note that the API differs from how we do it in the mobile SDKs @RevenueCat/coresdk. In those, we set this up as a static variable before `configure`. I felt this made more sense but happy to hear your thoughts